docs/reference/ImageDraw.rst

```diff
@@ -212,8 +212,8 @@ Methods
         .. versionadded:: 1.1.5
 
         .. note:: This option was broken until version 1.1.6.
-    :param joint: Joint type between a sequence of lines. It can be "curve",
-                  for rounded edges, or None.
+    :param joint: Joint type between a sequence of lines. It can be "curve", for rounded
```

Comment:
It's a minor change, so just for my own knowledge - why bring this text back up to this line?

Reply:
Wrapping to fit the line length shown in the IDE:

![image](https://user-images.githubusercontent.com/1324225/83734728-0df95b80-a658-11ea-9d1e-c6c50dcaf301.png)

I just spotted "curve" should be `"curve"`, so I've updated it and some others in 8764e088b345047d6b22e46493435674b93f1652 and undone the line break.